qprofiler2/src/org/qcmg/qprofiler2/summarise/SampleSummary.java

```diff
@@ -23,158 +23,167 @@
 	
-	public final static String genotype = "Genotype";
-	public final static String report = "report";	
-	public final static String VAF = "variantAltFrequencyPercent";
-	public final static String tiTvRatio = "TiTvRatio";
-	public final static String transitions ="Transitions";
-	public final static String transversions = "Transversions";
-	public final static String dbSNP = "inDBSNP";
+	public static final String genotype = "Genotype";
+	public static final String report = "report";	
+	public static final String VAF = "variantAltFrequencyPercent";
+	public static final String tiTvRatio = "TiTvRatio";
+	public static final String transitions = "Transitions";
+	public static final String transversions = "Transversions";
+	public static final String dbSNP = "inDBSNP";
 		
-	//allele frequency value range. eg. altBinSize = 10, each bin contain counts for variants which mutation allele percent is [0, 0.1]
-	public final static int altBinSize = 20; 
+	// allele frequency value range. eg. altBinSize = 10, each bin contain counts for variants which mutation allele percent is [0, 0.1]
+	public static final int altBinSize = 20; 
 		
-	Set< String > gts = new HashSet<>(); //store possible genotyp 0/1, 0/0, ./1 ...
-	Map< String, AtomicLong > summary = new HashMap<>();
+	Set<String> gts = new HashSet<>();  // store possible genotyp 0/1, 0/0, ./1 ...
+	Map<String, AtomicLong> summary = new HashMap<>();
 	Map<String, QCMGAtomicLongArray> summaryAD = new HashMap<>();
 	
 	AtomicLong counts = new AtomicLong();
 
-	private void increment(String key){	summary.computeIfAbsent(key, v -> new AtomicLong()).incrementAndGet(); }
+	private void increment(String key) { 
+		summary.computeIfAbsent(key, v -> new AtomicLong()).incrementAndGet();
+	}
 	
 	/**
 	 * Updates the supplied map with 
 	 * NOT SIDE-EFFECT FREE
-	 * @param type
-	 * @param gt
-	 * @param ad
-	 * @param dp
-	 * @param map
+	 * @param type is the variant type
+	 * @param gt is the value of GT from VCF format field
+	 * @param ad is the value of AD from VCF format field
+	 * @param dp is the value of DP from VCF format field
+	 * @param map is the pair of additional field from VCF format field
 	 */
-	public static void incrementGTAD(SVTYPE type,String gt, String ad, String dp, Map<String, QCMGAtomicLongArray> map){	
+	public static void incrementGTAD(SVTYPE type,String gt, String ad, String dp, Map<String, QCMGAtomicLongArray> map) {
 		
-		if( ad == null || ad.contains(".") ||  gt == null || gt.contains(".")  ||  gt.equals("0/0") || gt.equals("0|0") ) 
+		if ( ad == null || ad.contains(".") ||  gt == null || gt.contains(".")  ||  gt.equals("0/0") || gt.equals("0|0") ) {
 				return;		
+		}
 		
 		int commaIndex = ad.indexOf(Constants.COMMA);  
 		int vaf = 0;
 		
 		/*
 		 *  vaf needs to equal the sum of all the numbers in the AD field, apart from the first number (which is the reference count)
 		 */
-		while (commaIndex > -1) {
+		while (commaIndex > -1) { 
```

Comment:
In several places this file has got a new trailing space after open `{` blocks that shouldn't be there

Reply:
I only fix issues on src, due to style check didn't check code under test.  No worries, I can fix it now.